docs/canvas/canvas-workpad.asciidoc

```diff
@@ -69,25 +69,68 @@ Each of the sample data sets comes with a Canvas workpad that you can use for yo
 
 . Add a {kibana-ref}/add-sample-data.html[sample data set].
 
-. On the *Add Data to Kibana* page, click the *View data* dropdown list, then select *Canvas*.
-+
-Need some more workpad inspiration? Check out the link:https://www.elastic.co/blog/[Elastic Blog].
+. On the *Add Data* page, click the *View data* dropdown list, then select *Canvas*.
 
 [float]
 [[apply-workpad-styles]]
 == Apply a set of styles to the entire workpad
 
-To make your workpad look exactly the way you want, use the editor on the right to apply CSS overrides.
+To make your workpad look exactly the way you want, use the editor to apply CSS overrides.
 
 . Expand *Global CSS overrides*.
 
-. Enter the CSS. For example, to change the background on every page, enter:
+. Enter the CSS.
++
+For example, to change the background on every page, enter:
 +
-[source,js]
+[source,text]
 --------------------------------------------------
 .canvasPage {
 background-color: #3990e6;
 }
 --------------------------------------------------
 
 . Click *Apply stylesheet*.
+
+[float]
+[[configure-auto-refresh-interval]]
+== Change the auto-refresh interval
+
+Increase or decrease how often the data refreshes on your workpad.
+
+. In the top left corner, click the *Control settings* icon.
+
+. Under *Change auto-refresh interval*, select the interval you want to use.
++
+[role="screenshot"]
+image::images/canvas-refresh-interval.png[Element data refresh interval]
+
+TIP: To manually refresh the data, click the *Refresh data* icon.
+
+[float]
+[[zoom-in-out]]
+== Use the zoom options
+
+In the upper left corner, click the *Zoom controls* icon, then select one of the options.
+
+[role="screenshot"]
+image::images/canvas-zoom-controls.png[Zoom controls]
+
+[float]
+[[add-more-pages]]
+== Add pages
+
+Add more pages when you've run out of room, or for when you'd like to separate your ideas across several pages.
```

Comment:
I'd switch and put the idea about separating your ideas first.  And maybe its more like separating your ideas into digestable chunks.